MSET9_installer_script/mset9.py

```diff
@@ -71,8 +70,46 @@ def remove_extra():
 
 osver = platform.system()
 thisfile = os.path.abspath(__file__)
+scriptroot = os.path.dirname(thisfile)
 systmp = None
 
+# I hate python ternary operator i hate python ternary operator
+systemroot = os.environ["SYSTEMDRIVE"] if osver == "Windows" else "/" # Never hardcode C:. My Windows drive letter is E:, my SD card or USB drive is often C:.
```

Comment:
did this get tested for users with Msys2 Non-mingw/Cygwin python? Although unlikely most common users will use it, but still, these two can use / paths, like /c/ for C:\

Reply:
@luigoalma Using `os.name == "posix"` will fix that

Reply:
@Anonymous941 
You just have to pull request to the head branch of the PR - https://github.com/Naim2000/MSET9/tree/fresh
Feels kind of self promo but I don't want to ctrl+C V your suggestion without (personally) knowing what it does

Reply:
@Naim2000

> You just have to pull request to the head branch of the PR

Got it, I've made a PR in Naim2000/MSET9#1

> Feels kind of self promo

Wdym?

> I don't want to ctrl+C V your suggestion without (personally) knowing what it does

[Here's the documentation page](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.anchor) - basically it just adds the system drive (`C:` for Windows, blank for UNIX) and root (aka `/` or `\`).  On Windows this is `C:\`, UNIX is `/` and Cygwin is `C:/`, so it does what you need

Reply:
Thank you
> Wdym?

I think the word I was looking for was 'lazy'. But then again, I don't like throwing in suggestions without 100% knowing how it works. So thank you for the docs as well